custom_dc/deploy_cloud.md

```diff
@@ -52,68 +52,65 @@ This procedure creates a "dev" Docker package that you upload to the Google Clou
-
-## Deploy the image
+1. In the list of repositories, click on the one you created above. Under **Repository Details**, you should see the Docker image listed.
 
-To deploy the image in Google Cloud Run, you need to create a Run service. Because we need to set the environment variables for the running image as options in the service, it is actually more convenient to create the service and deploy the image at the same time using `gcloud` rather than the Cloud Console. Once the service is created in this way, you can edit it and redeploy using the Console.
+## One-time setup: Create a Google Run service
 
-1. Copy the settings from the `env.list` file to a local environment variable:
-
-   ```shell
-   env_vars=$(awk -F '=' 'NF==2 {print $1"="$2}' custom_dc/env.list | tr '\n' ',' | sed 's/,$//')
-   ```
-
-1. Create a new Cloud Run service and deploy the image in the Artifact Registry.
-
-   <pre>
-   gcloud run deploy <var>SERVICE_NAME</var> \  
-   --allow-unauthenticated \  
-   --memory 8Gi \  
-   --cpu 2 \  
-   --region <var>LOCATION</var> \  
-   --image <var>LOCATION</var>-docker.pkg.dev/<var>PROJECT_ID</var>/<var>ARTIFACT_REPO</var>/<var>IMAGE_NAME</var>:<var>TARGET_IMAGE_TAG</var>  \
-   --add-cloudsql-instances=<var>PROJECT_ID</var>:<var>LOCATION</var>:<var>INSTANCE_ID</var> \  
-   --set-env-vars="$env_vars" \  
-   --port 8080  
-   --no-cpu-throttling
-   </pre>
-
-   The _SERVICE_NAME_ can be any string of your choice.
-
-1. Follow the screen output to see the status details of the operation. Once it completes, a link to the deployed image is output. Visit the link in your browser to see the running instance.
-
-To inspect the service in the Cloud Console:
-
-1. Go to [https://console.cloud.google.com/run](https://console.cloud.google.com/run) for your project.
-1. Select the service you just created to show the **Service Details** page.
-1. Select **Revisions**. Scroll down and you can see the environment variables and other settings.
-
-For future deployments, select **Edit & Deploy Revision** from this page to select a new repository\ container and locate the link to the running instance.
+This procedure shows you how to create a service and set environment variables using the Cloud Console. 
 
 See also [Deploying to Cloud Run](https://cloud.google.com/run/docs/deploying) for more information on all the options you may set on your service.
 
-## Load custom data to a deployed instance
-
-Once you have deployed a custom instance to Google Cloud, you can continue to update your custom data in two ways:
-
-- Load the data from a local running instance, as described in [Load custom data in Cloud SQL](/custom_dc/data_cloud.html#load-data-cloudsql)
-- Use the `/admin` page from the running Cloud app.
+1. Go to https://console.cloud.google.com/run/ for your project.
+1. Click **Create service**.
```

Comment:
It may be useful to include screenshots for some of these.

Reply:
Yeah, you're probably right. I try to avoid it because they constantly change the UI. But I will add one for these detailed ones.

Reply:
Done.